src/selectors/per-thread/thread.js

```diff
@@ -340,11 +344,25 @@ export function getThreadSelectorsPerThread(
     ProfileData.getThreadProcessDetails
   );
 
-  const getTransformLabels: Selector<string[]> = createSelector(
+  const getTransformLabelL10nIds: Selector<
+    TransformLabeL10nIds[]
+  > = createSelector(
     getRangeAndTransformFilteredThread,
     getFriendlyThreadName,
     getTransformStack,
-    Transforms.getTransformLabels
+    Transforms.getTransformLabelL10nIds
+  );
+
+  const getLocalizedTransformLabels: Selector<
+    Localized[]
+  > = createSelector(getTransformLabelL10nIds, transformL10nIds =>
+    transformL10nIds.map((transform, idx) => (
+      <Localized
+        id={transform.l10nId}
+        vars={{ item: transform.item }}
+        key={`${transform.l10nId}${idx}`}
```

Comment:
I think that instead of `idx` you should use item here. Indeed the couple (l10nid + item) represents exactly this item.

but this is not a big deal in this case because this is a trivial element anyway, and we never reorder them, so up to you!